mpyl_config.example.yml

```diff
@@ -169,12 +176,23 @@ project: # default values
       enabled: true
       hosts:
         - host:
-            pr: "Host(`{SERVICE-NAME}-{PR-NUMBER}.test-backend.nl`)"
-            test: "Host(`{namespace}-{SERVICE-NAME}.test-backend.nl`)"
-            acceptance: "Host(`{namespace}-{SERVICE-NAME}.acce-backend.nl`)"
-            production: "Host(`{namespace}-{SERVICE-NAME}.prod-backend.nl`)"
+            pr: "Host(`{SERVICE-NAME}-{PR-NUMBER}.{CLUSTER-ENV}-backend.nl`)"
+            test: "Host(`{namespace}-{SERVICE-NAME}.{CLUSTER-ENV}-backend.nl`)"
+            acceptance: "Host(`{namespace}-{SERVICE-NAME}.{CLUSTER-ENV}-backend.nl`)"
+            production: "Host(`{namespace}-{SERVICE-NAME}.{CLUSTER-ENV}-backend.nl`)"
           tls:
             all: "le-custom-prod-wildcard-cert"
           insecure: false
           whitelists:
             all: [ "VPN" ]
+    additionalTraefikRoutes:
+      - name: "ingress-intracloud-https"
+        clusterEnv:
+          pr: "test-other"
+          test: "test-other"
+          acceptance: "acce-other"
+          production: "acce-other"
+        middlewares:
+          - "intracloud-middleware@kubernetescrd"
+        entrypoints:
+          - "intracloud"
```

Comment:
Random note: I think I've seen @ptab mention somewhere that having a newline at the end of your files should be the default